client/main/notifications.js

```diff
@@ -1,19 +1,37 @@
 
 'use strict';
 
-// const Notification = require('./notification');
+var $ = require('jquery');
+
+const NotificationView = require('./notification');
 
 const Notifications = function($el) {
     this.$el = $el;
-    this._requests = [ ];
+    this._list = [ ];
 };
 
 Notifications.prototype.notify = function(request) {
 
     console.log('notification received');
     console.log(request.attributes);
+    
+    var $note = $('<div id="notification"></div>');
+    this.$el.append($note);
+    request.set("index", this._list.length);
+    request.parent = this;
```

Comment:
Nah, I didn't catch that I'll amend

On 22/08/2016 12:07 pm, "Jonathon Love" <notifications@github.com> wrote:

> In client/main/notifications.js
> <https://github.com/silkyproject/silky/pull/52#discussion_r75610512>:
>
> >  };
> >
> >  Notifications.prototype.notify = function(request) {
> >
> >      console.log('notification received');
> >      console.log(request.attributes);
> > +
> > +    var $note = $('<div id="notification"></div>');
> > +    this.$el.append($note);
> > +    request.set("index", this._list.length);
> > +    request.parent = this;
>
> is request.parent still needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/silkyproject/silky/pull/52/files/e3ac9a31dd808a6be45e8a640b6b7da4146bf77b#r75610512>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AR4csTijDMiYb0dspIPMggRcvnVePiTlks5qiQRDgaJpZM4JpNpx>
> .
>